src/base/bittorrent/session.cpp

```diff
@@ -724,6 +770,45 @@ void Session::setSubcategoriesEnabled(bool value)
     emit subcategoriesSupportChanged();
 }
 
+const QSet<QString> &Session::tags() const
```

Comment:
Thanks for pointing that out. I read up on Qt's containers after reading your comments.